static/src/javascripts/projects/common/modules/identity/validation-email.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import mediator from 'lib/mediator';
+import IdentityApi from 'common/modules/identity/api';
+
+const init = (): void => {
+    const resendButton = document.getElementsByClassName(
```

Comment:
fastdom? 👀

Reply:
Ah yes, good suggestion, I should get into the habit of using fastdom.read/write when querying/writing to the DOM.